docs/adr/README.md

```diff
@@ -0,0 +1,16 @@
+# Architecture Decision Records
+
+This folder keeps all important architecture decisions for Provisioning service backend.
+ADR is a well established process in software engineering and more details can be found at [https://adr.github.io/](https://adr.github.io/).
+
+### ADR lifecycle management
+
+* **Key decider** - person driving the decision
+* **Stakeholder** - everyone affected by the decision
+
+1. ADR is drafted in Google doc that is based on [Template](adr/00000-template.md) by key decider
+2. Decision is proposed to all stakeholders by key decider
+3. Decide by is set after initial discussion, or the topic is dropped if not deemed worthy to pursue.
+4. Key decider opens a PR with the Final draft of decision.
+5. ADR is merged in Proposed state (Draft is only for gdocs)
+6. The PR implementing the decision will change the status to Accepted
```

Comment:
I think this is way too complex, can't we just simplify this to:

* Draft ADR = open PR
* Accepted = merged

I mean if anyone wants to write their ADRs in MS Word or whatever that is fine. Please let’s KISS. :-)

Reply:
Agreed, this was copied from somewhere, I believe you can have Proposed one for discussion before implementation, but only as optional. I'll simplify this.